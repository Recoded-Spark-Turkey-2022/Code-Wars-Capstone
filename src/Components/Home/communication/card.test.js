
import React from "react";
import renderer from "react-test-renderer";
import Card from "./card";
import { MemoryRouter } from "react-router-dom";

it("renders correctly", () => {
    const test = renderer
        .create(
            <MemoryRouter>
                <Card/>
            </MemoryRouter>
        )
        .toJSON();
    expect(test).toMatchSnapshot();
});