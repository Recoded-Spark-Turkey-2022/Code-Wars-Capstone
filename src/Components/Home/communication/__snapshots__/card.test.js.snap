// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-white rounded-lg drop-shadow-xl p-6 w-10/12"
  style={
    Object {
      "fontFamily": "Poppins, sans-serif",
    }
  }
>
  <img
    alt="cardsphoto"
    className="h-20 w-15 rounded-full mx-auto mb-4"
  />
  <h2
    className="text-2xl font-bold text-center mb-2"
  />
  <p
    className="text-gray-700 text-center"
  />
</div>
`;
